src/main/java/com/users/entities/User.java

```diff
@@ -24,7 +26,17 @@ public class User extends AuditorEntity {
             regexp = "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,6}$",
             message = "Invalid email format"
     )
-    private String email;
+    private String primaryEmail;
+
+    @ValidEmailList
+    private List<String> secondaryEmail;
```

Comment:
ai incercat sa salvezi in db? cum arata?

Reply:
am incercat, validarea e buna, dar am problema aia cand adaug unul nou, aia peste care vreau sa ne uitam